components/02-content/jumbotron/jumbotron.njk

```diff
@@ -2,15 +2,15 @@
   <h2 class="dso-jumbotron-title">{{ title }}</h2>
   <div class="dso-jumbotron-body">
     <p>{{ content }}</p>
-    {{#if button }}
+    {%- if button %}
```

Comment:
Is de whitespace before control bewust? In de handlebars stond er geen `~`

Reply:
Yep, er verschijnt via nunjucks toch een whitespace die we daar weg willen hebben